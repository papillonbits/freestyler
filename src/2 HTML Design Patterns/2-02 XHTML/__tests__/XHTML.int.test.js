import { regular } from './XHTML.int.story'

describe('<XHTML />', () => {
  describe('Render', () => {
    test('must match regular()', () => {
      expect(global.renderToJSON(regular())).toMatchSnapshot()
    })
  })
})
