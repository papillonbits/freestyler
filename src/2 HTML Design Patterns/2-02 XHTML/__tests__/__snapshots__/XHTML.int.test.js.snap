// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<XHTML /> Render must match regular() 1`] = `
<div>
  <div
    class="xhtml"
  >
    <h1>
      XHTML
    </h1>
     
    <p>
      Paragraph
    </p>
     
    <br />
    Break
    <ol>
       
      <li>
        Ordered List Item
      </li>
       
      <li>
        Ordered List Item
      </li>
       
    </ol>
    <dl>
       
      <dt>
        Definition Term
      </dt>
       
      <dd>
        Definition Data
      </dd>
       
    </dl>
    <iframe
      src="html_example.html"
      title="html-example"
    />
  </div>
</div>
`;
